src/lib/uni/src/unified.ts

```diff
@@ -0,0 +1,200 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {
+  AfterContentInit,
+  ContentChildren,
+  Directive,
+  ElementRef,
+  Inject,
+  Input,
+  OnDestroy,
+  Optional,
+  QueryList,
+  SkipSelf,
+} from '@angular/core';
+
+import {GrandCentral} from './central';
+import {BPS, Breakpoint, FALLBACK_BREAKPOINT, FALLBACK_BREAKPOINT_KEY} from './breakpoint';
+import {Tag, ValuePriority} from './tags/tag';
+import {TAGS} from './tags/tags';
+import {ATTR_KEY, CLASS_KEY, KEY_DELIMITER, STYLE_KEY, CLEAR_VALUE} from './constants';
+
+
+/**
+ * This is a simplistic, wrapping directive meant only to
+ * capture and record values for individual breakpoints.
+ */
+@Directive({selector: `bp`})
+export class BreakpointDirective {
+
+  @Input('tag') name: string = '';
+
+  readonly element: HTMLElement;
+  constructor(elementRef: ElementRef<HTMLElement>) {
+    this.element = elementRef.nativeElement;
+  }
+}
+
+/**
+ * One directive to rule them all. This directive is responsible for
+ * tagging an HTML element as part of the layout system, and then
+ * coordinating all updates with GrandCentral.
+ */
+@Directive({selector: `[ngl]`})
```

Comment:
This seems like a [structural directive](https://angular.io/guide/structural-directives) which should have a `*` in front of it (when used in a template), since it can change the structure of the DOM (especially when a directive like `order="2"` is used).

But then that introduces the limitation of [one structural directive per host element](https://angular.io/guide/structural-directives#one-structural-directive-per-host-element).

Have you considered this approach? Were issues uncovered?